src/lib.rs

```diff
@@ -314,6 +314,33 @@ where
         Ok(())
     }
 
+    /// Set mirror/vertical flip.
+    pub fn set_mirror(&mut self, mirror: bool) -> Result<(), DisplayError> {
+        if mirror {
+            match self.rotation {
+                DisplayRotation::Rotate0 => {
+                    Command::SegmentRemap(false).send(&mut self.interface)?;
+                    Command::ReverseComDir(true).send(&mut self.interface)?;
+                }
+                DisplayRotation::Rotate90 => {
+                    Command::SegmentRemap(true).send(&mut self.interface)?;
+                    Command::ReverseComDir(true).send(&mut self.interface)?;
+                }
+                DisplayRotation::Rotate180 => {
+                    Command::SegmentRemap(true).send(&mut self.interface)?;
+                    Command::ReverseComDir(false).send(&mut self.interface)?;
+                }
+                DisplayRotation::Rotate270 => {
+                    Command::SegmentRemap(false).send(&mut self.interface)?;
+                    Command::ReverseComDir(false).send(&mut self.interface)?;
+                }
```

Comment:
In my testing, I found the 270º and 90º configs should be swapped:

```suggestion
            match self.rotation {
                DisplayRotation::Rotate0 => {
                    Command::SegmentRemap(false).send(&mut self.interface)?;
                    Command::ReverseComDir(true).send(&mut self.interface)?;
                }
                DisplayRotation::Rotate90 => {
                    Command::SegmentRemap(false).send(&mut self.interface)?;
                    Command::ReverseComDir(false).send(&mut self.interface)?;
                }
                DisplayRotation::Rotate180 => {
                    Command::SegmentRemap(true).send(&mut self.interface)?;
                    Command::ReverseComDir(false).send(&mut self.interface)?;
                }
                DisplayRotation::Rotate270 => {
                    Command::SegmentRemap(true).send(&mut self.interface)?;
                    Command::ReverseComDir(true).send(&mut self.interface)?;
                }
            };
```

Did you see different results on your display? Can you apply this suggestion and retest on your end? Hopefully we don't have different behaviour!